packages/locale/webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+console.log(path.resolve('./dist'));
+
+module.exports = {
+	mode: 'production',
+	entry: {
+		index: './src/locale.js',
+	},
+	output: {
+		filename: '[name].js',
+		path: path.resolve('./dist'),
+		libraryTarget: 'umd',
+	},
+	module: {
+		rules: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				loader: 'babel-loader',
```

Comment:
rootMode forgotten again

Reply:
It's not necessary in this build, but it's true that we should have same config across builds